packages/service-clients/odsp-client/src/odspAudience.ts

```diff
@@ -3,21 +3,42 @@
  * Licensed under the MIT License.
  */
 import { assert } from "@fluidframework/core-utils";
-import { type IClient } from "@fluidframework/protocol-definitions";
+import { IUser, type IClient } from "@fluidframework/protocol-definitions";
 
-import { type OdspMember, OdspUser } from "./interfaces";
+import { type OdspMember } from "./interfaces";
+
+/**
+ * Since ODSP provides user names and oids for all of its members, we extend the
+ * {@link @fluidframework/protocol-definitions#IMember} interface to include this service-specific value.
+ * @internal
+ */
+interface OdspUser extends IUser {
+	/**
+	 * This is the email id of the user
+	 * Overrides the documentation from IUser.id.
+	 */
+	id: string;
+	/**
+	 * This is the name of the user
+	 */
+	name: string;
+	/**
+	 * The object ID or object Identifier. It is a unique identifier assigned to each user, group, or other entity within AAD or another Microsoft 365 service. It is a GUID that uniquely identifies the object. When making Microsoft Graph API calls, you might need to reference or manipulate objects within the directory, and the `oid` is used to identify these objects.
+	 */
+	oid: string;
```

Comment:
Would recommend using `userId` here to match with `IMember` and have fewer member names to remember.

Reply:
this is just an internal type, its not exposed, its for mapping the returned iuser to a imember